packages/console/src/pages/Applications/components/TypeDescription/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  subtitle: string;
+  description: string;
+};
+
+const TypeDescription = ({ subtitle, description }: Props) => {
+  return (
+    <>
```

Comment:
nit:  Sometimes we intend to give a top-level empty div wrapper for those widely used common UI components. Just to keep them layout-safe.  As the upper sope consumers' styles may vary, i.e display flex.  Without an actual div container, the layout may breaks.  
But it truly depends on the usage cases.

Reply:
agreed, since it's more like a business component thus i use fragment